src/main.rs

```diff
@@ -170,7 +182,18 @@ async fn run_cql_query(
 
     let mut key_exists = false;
 
-    let report_from_cache = match report_cache.lock().await.cache.get(encoded_query) {
+    let mut obfuscate = false;
+
+    if CONFIG.obfuscate == config::Obfuscate::Yes && !CONFIG.unobfuscated.contains(&project) {
+        obfuscate = true;
+    }
```

Comment:
This can be written like so no?
```suggestion
    let obfuscate = CONFIG.obfuscate == config::Obfuscate::Yes && !CONFIG.unobfuscated.contains(&project);
```